pinot-plugins/pinot-input-format/pinot-protobuf/src/test/java/org/apache/pinot/plugin/inputformat/protobuf/ProtoBufRecordExtractorTest.java

```diff
@@ -166,7 +166,7 @@ private Map<String, Object> createRecord1() {
     record.put(LONG_FIELD, 100L);
     record.put(DOUBLE_FIELD, 1.1);
     record.put(FLOAT_FIELD, 2.2f);
-    record.put(BOOL_FIELD, "false");
+    record.put(BOOL_FIELD, "true");
```

Comment:
Why was this changed from `false` to `true`?

Reply:
Taking a look at the test code, if we change this to `true` then none of the tests will verify that we correctly parse a Protobuf message with a boolean field whose value is `false`.

Reply:
Yes I thought about this as well.  Protobuf seems to treat the field as optional, when the value is set to its default. This seems to be the behavior. Whats your suggestion to handle this ?  I found a similar issue in the community. 
https://github.com/protocolbuffers/protobuf/issues/10560

Reply:
> Yes I thought about this as well. Protobuf seems to treat the field as optional, when the value is set to its default. This seems to be the behavior. Whats your suggestion to handle this ? I found a similar issue in the community. [protocolbuffers/protobuf#10560](https://github.com/protocolbuffers/protobuf/issues/10560)

Reply:
To make sure I'm not missing anything: if we set a boolean field (which is not optional) to `false` then the PB API will say that it is optional?

Reply:
> To make sure I'm not missing anything: if we set a boolean field (which is not optional) to `false` then the PB API will say that it is optional?

Yes thats correct. Thats what I observed and also read from other sources.

Reply:
Ok, then we should have a separate test to confirm the optional that uses a different message, that way we can properly test that we extract `false` from a message.